helm/dashboards/templates/configmap.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   name: grafana-dashboards-public-shared
   namespace: {{ .Values.namespace  }}
 data:
+  home.json: {{ (tpl (.Files.Get "dashboards/home.json") . ) | quote }}
```

Comment:
@marians this line makes ensure that helm templates the `home.json` dashboard instead of just rendering it as is so we can use helm values in there :)